index.js

```diff
@@ -164,6 +164,9 @@ function SpotifyResolve(createOpts) {
       if (error) {
         done(error);
       }
+      else if (response.statusCode === 401) {
+        done(new Error(response.statusCode), response);
```

Comment:
I was hoping to send both a more verbose error message as well as the statusCode since I want to have a good way to let `this-is-an-editor` know that this particular error is a 401 so it can handle it appropriately 👇  instead of just displaying the `error.stack` to the user. 
`
if (Number(error.message) === 401) {
   redirectToAuth(window.location.hash.slice(1));
 }
`
But I also created a PR for an alternative solution where instead of redirecting immediately when a 401 error is received, we check the token expiration. So I can make this update if the latter is the better solution :)